next-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
typescript boilerplate

decided to add typescript to the project. It's non binding and opt-in, so JS files should compile just fine as well